frontend/src/metabase/query_builder/components/DataSelector/DataSelector.unit.spec.js

```diff
@@ -418,7 +421,7 @@ describe("DataSelector", () => {
     ).toBeInTheDocument();
   });
 
-  it("should move between selected multi-schema dbs", () => {
+  it("should move between selected multi-schema dbs", async () => {
```

Comment:
It's not clear why we made this (And line 405) async

Reply:
Also, there are some occurrences of fireEvent in these tests

Reply:
honestly it's just annoying to have to change it to async when you want to await. i'd love to introduce a lint rule to force folks to always using async callbacks for test fns.